src/content-helper/editor-sidebar/related-posts/hooks.ts

```diff
@@ -36,13 +37,31 @@ interface PostData {
  *
  * @since 3.14.3
  * @since 3.14.4 Implemented checks to reduce risk of invalid data being processed.
+ * @since 3.16.2 Added a timeout to ensure the hook returns after X seconds.
```

Comment:
Maybe change "seconds" to "milliseconds" as this is what the function expects?

Reply:
Good call!